lib/src/flutter/code_generation/generators/gherkin_suite_test_generator.dart

```diff
@@ -70,14 +81,14 @@ void executeTestSuite(
 
     final featureExecutionFunctionsBuilder = StringBuffer();
     final generator = FeatureFileTestGenerator();
-    final featuresToExecute = new StringBuffer();
+    final featuresToExecute = StringBuffer();
     var id = 0;
 
-    for (var featureFileContent in featureFiles) {
+    for (var featureFile in featureFiles) {
       final code = await generator.generate(
         id++,
-        featureFileContent,
-        '',
+        featureFile.readAsStringSync(),
+        featureFile.absolute.path,
```

Comment:
@AFASbart having an absolute path is not optimal, as for users who save the file in git to use it for Github actions (like we do) the absolute path is not right / contains personal information...

Reply:
Agree

Reply:
I suppose the downside here is that you cannot jump to the file in the console with a relative path

Reply:
@golane-august see the release note for 3.0,0-rc-11 as I've added an option to generate relative paths